gctscript/wrappers/gct/gctwrapper_test.go

```diff
@@ -249,16 +242,11 @@ func TestExchangeOrderSubmit(t *testing.T) {
 		orderAmount,
 		orderID,
 		orderAsset)
-	if err != nil {
-		t.Fatalf("received: %v but expected: %v", err, nil)
-	}
+	require.NoError(t, err)
 
-	rString, _ := objects.ToString(obj)
-	if rString != `error: "Bitstamp REST or Websocket authentication support is not enabled"` {
-		t.Errorf("received: [%v] but expected: %v",
-			rString,
-			`error: "Bitstamp REST or Websocket authentication support is not enabled"`)
-	}
+	rString, ok := objects.ToString(obj)
+	require.True(t, ok)
```

Comment:
🌴🐒 This is the same nitpick monkey as above. It just moved trees to repeat itself 😄